tests/test_cli.py

```diff
@@ -0,0 +1,213 @@
+import csv
+import itertools
+import json
+import subprocess
+import tempfile
+
+import pytest
+
+
+CMD_ROOT = "browser-history"
+
+# Options:
+VALID_CMD_OPTS = [
+    (f"--{long_o}", f"-{short_o}")
+    for long_o, short_o in [
+        ("help", "h"),
+        ("type", "t"),
+        ("browser", "b"),
+        ("format", "f"),
+        ("output", "o"),
+    ]
+]
+INVALID_CMD_OPTS = [
+    "--history",
+    "-a",
+    "-B",
+    "-0",
+]
+
+# Arguments, where the default arg should be put first for default testing:
+VALID_TYPE_ARGS = [
+    "history",
+    "bookmarks",
+    # "all",  # Note: this is only commented out because there is a bug!
```

Comment:
In writing this test module, I believe I have discovered a bug in the CLI. Namely, `all` is specified as a valid keyword accepted by the `--type` option in the command help description, but it does not work at the moment; I uncomment this line so that the command `browser-history --type all` gets run and tested, it leads to a test failure.

On the CLI if I try it directly I see:
```bash
Traceback (most recent call last):
  File "/home/sadie/anaconda3/envs/cf-env/bin/browser-history", line 33, in <module>
    sys.exit(load_entry_point('browser-history', 'console_scripts', 'browser-history')())
  File "/home/sadie/browser-history/browser_history/cli.py", line 95, in main
    assert (
AssertionError: Type should be one of all, history, bookmarks
```

@Samyak2  shall I raise this as a separate issue or shall I fix it as part of this PR (I am happy to do so)?

Reply:
For me (on the master branch), the `--help` page does not show `all` as a valid type and I get the following error when I try it.
```
❯ browser-history --type all
ERROR: Type all is unavailable. Check --help for available types
```

Reply:
Ah yes, that's true, the help must have changed since, since now on master I also see:

```console
$ browser-history --help
...
  -t TYPE, --type TYPE  argument to decide whether to retrieve history or
                        bookmarks. Should be one of history, bookmarks.
                        Default is history.
...
```

therefore I guess that option was removed since, or similar. So it seems I just need to remove this commented line. I'll do that in the next commit.

Reply:
Ah yes, that's true, the help must have changed since, since now on master I also see:

```console
$ browser-history --help
...
  -t TYPE, --type TYPE  argument to decide whether to retrieve history or
                        bookmarks. Should be one of history, bookmarks.
                        Default is history.
...
```

therefore I guess that option was removed since, or similar. So it seems I just need to remove this commented line. I'll do that in the next commit.